### PR TITLE
Honor bundles configuration in controller injectors warmer

### PR DESCRIPTION
**Pull request by gremo**
This should solve #269 and #248.

### CODE REVIEW

**Comment by greg0ire** [comment]
ping @GuilhemN please review

**Comment by GuilhemN** [comment]
Sorry I won't have the time to review this. Please do and if you approve it, I'll merge it.

File: HttpKernel/ControllerInjectorsWarmer.php [hunk 1]
```diff
@@ -28,12 +28,21 @@ class ControllerInjectorsWarmer implements CacheWarmerInterface
...
+    private $scanAllBundles;
+    private $scanBundles;
 
-    public function __construct(KernelInterface $kernel, ControllerResolver $resolver, array $blackListedControllerFiles)
-    {
+    public function __construct(
+        KernelInterface $kernel,
+        ControllerResolver $resolver,
+        array $blackListedControllerFiles,
+        $scanAllBundles,
```
Comment (greg0ire) [inline HttpKernel/ControllerInjectorsWarmer.php]: Should be optional and default to `true`

File: HttpKernel/ControllerInjectorsWarmer.php [hunk 2]
```diff
@@ -28,12 +28,21 @@ class ControllerInjectorsWarmer implements CacheWarmerInterface
...
+        array $scanBundles
```
Comment (greg0ire) [inline HttpKernel/ControllerInjectorsWarmer.php]: Should be optional and default to `array()`. That way, the constructor is still callable like it was before.

Comment (gremo) [reply HttpKernel/ControllerInjectorsWarmer.php]: I'm going to push the changes. Out of curiosity, why provinding default values just to make the constructor callable like before? Service is created and managed by Symfony DI, so why bother?

Comment (greg0ire) [reply HttpKernel/ControllerInjectorsWarmer.php]: Because who knows who might use it without sf DI?

**Comment by OskarStark** [comment]
Travis failed 😢

**Comment by greg0ire** [comment]
@OskarStark it failed before and is apparently not a blocker.

**Comment by gremo** [comment]
Is there something I can do to get this merged?

**Comment by greg0ire** [comment]
Pray your god and/or @GuilhemN

**Comment by GuilhemN** [comment]
> Travis failed 😢

Yes, we have to fix them but i'm lacking time...

> Pray your god and/or @GuilhemN

Ahah 😄 I'd be glad to you as collaborator since I can't put much time in this repository but unfortunately I don't have the admin rights 😕

**Comment by GuilhemN** [comment]
Thank you @gremo and @greg0ire !

**Comment by OskarStark** [comment]
@GuilhemN can you create a release?

**Comment by greg0ire** [comment]
I don't know much about this bundle or even use it, but that should not stop me from providing some help if you so desperately need it :) I'll try to have a look at pending PRs

**Comment by greg0ire** [comment]
Please don't tag yet, this seems to be buggy: #287

**Comment by GuilhemN** [comment]
> I don't know much about this bundle or even use it, but that should not stop me from providing some help if you so desperately need it :) I'll try to have a look at pending PRs

I'm in a french "prépa" since the beginning of the month so I'm really running out of time 😄 thanks for taking care of this bundle, but of course don't feel forced to do anything ☺ 

@schmittjoh could you add @greg0ire as collaborator please?

**Comment by greg0ire** [comment]
> I'm in a french "prépa" 

I went through that myself! See you in 2 / 3 years then I guess :P
Good luck with it!

**Pull request merged**